agent_api_rest/src/verification/relying_party/redirect.rs

```diff
@@ -187,6 +187,9 @@ pub mod tests {
         request(&mut app, state.clone()).await;
         redirect(&mut app, state).await;
 
+        // Wait for the request to arrive at the mock server endpoint.
+        std::thread::sleep(std::time::Duration::from_millis(10));
```

Comment:
Is `10ms` safe for that test under all circumstances? Why not increase it to `100ms` to be even more certain?